src/main/webapp/js/user-page-loader.js

```diff
@@ -32,16 +32,19 @@ function setPageTitle() {
 /**
  * Shows the message form if the user is logged in and viewing their own page.
  */
-function showMessageFormIfViewingSelf() {
-  document.getElementById('about-me-form').classList.remove('hidden');
+
+ //Show the form as long as the user is logged in, even if they aren't viewing their own page
```

Comment:
Same here - the documentation is above, starting with `/**` and ending with `*/`.